mint-auditor/src/db.rs

```diff
@@ -439,8 +505,8 @@ mod tests {
 
         // Sync a block that mixes burning and minting.
         let mint_tx1 = create_mint_tx(token_id1, &signers1, 1000, &mut rng);
-        let mint_tx2 = create_mint_tx(token_id2, &signers1, 2000, &mut rng);
-        let mint_tx3 = create_mint_tx(token_id3, &signers1, 20000, &mut rng);
+        let mint_tx2 = create_mint_tx(token_id2, &signers2, 2000, &mut rng);
```

Comment:
I'm sure I'm misunderstanding something here, but these needed to be changed to valid signers in order for the `num_mint_txs_without_matching_mint_config` counter to be 0.
These look like burn transactions, should they be counting as mint transactions?

Reply:
NVM I misunderstood the test.
These are the mint transactions that to verify the final sum after minting and burning

Reply:
Correct. A burn transaction is a regular mobilecoin transaction that moves tokens into a specific address known as the burn address. This is an address that has a publicly known view key (so anyone can view-key scan it and see how much was sent to it) but no known spend key (so tokens going to that address can never be spent, and as such are no longer usable and effectively considered burnt).